internal/utils/utils.go

```diff
@@ -0,0 +1,84 @@
+package utils
```

Comment:
I need to find a better name as this is not recommended in Golang community

Reply:
We will gradually remove this package as it's not required